src/v2/mod.rs

```diff
@@ -776,7 +785,7 @@ fn extract_metadata<T>(response: &tonic::Response<T>) -> endpoints::RPCResult<Bl
 ///
 /// The main reason for needing this is that in proto3 all fields are optional,
 /// so it is up to the application to validate inputs if they are required.
-trait Require<E> {
+pub trait Require<E> {
```

Comment:
And neither should this. This has a very specific utility, and we don't want to expose it.